cross/build-android-rootfs.sh

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env bash
```

Comment:
@qmfrederik  ~~Thanks for the reminder; I had it in the issues checklist but forgot to merge pre-update.Will tackle this later today.~~

Just did a quick once-over of the script and had a few questions before I make changes:

1. Script still seems to build for arm64 only; did I miss something?
2. Is ARM support complete for coreclr/corefx?
3. Has NDK 14 builds of coreclr/corefx been tested on Android?